WorkloadManagementSystem/Service/JobMonitoringHandler.py

```diff
@@ -242,6 +242,14 @@ def export_getJobLoggingInfo( jobID ):
     return gJobLoggingDB.getJobLoggingInfo( jobID )
 
 ##############################################################################
+  types_getJobsParameters = [ ListType, ListType ]
+  @staticmethod
+  def export_getJobsParameters ( jobIDs, parameters ):
+    if not ( jobIDs and parameters ) : 
+      return S_OK( {} )
+    return gJobDB.getAttributesForJobList( jobIDs, parameters )
+
+##############################################################################
```

Comment:
Up to you, I was following the spelling of the other functions, e.g. we have a function for getting the Status of multiple jobs which is called "getJobsStatus" and similar for other single parameters. 

On the other hand functions which only query one or multiple parameters of a job start with "getJob<parameter>". 

That was my reasoning for calling the function like that.